san/sanbase_graphql.py

```diff
@@ -18,12 +13,6 @@ def burn_rate(idx, slug, **kwargs):
     return query_str
 
 
-def transaction_volume(idx, slug, **kwargs):
```

Comment:
only those 2 should be deleted? They are the ones with the 1:1 names, but mvrv, nvt, circulation, burn rate, etc. are also present in getMetric

Reply:
Maybe in another PR we need to just point the old names to the new names?

Reply:
Yeah, I was thinking of doing a PR, in which we just move from the old functions to the new ones, or just point them to the new functions, as you said.

Reply:
I was thinking, that this PR is for the 1:1 functions and the description change.

Reply:
I'll note the other PR idea, which we've just discussed.